bc_obps/registration/tests/endpoints/v1/_operations/_operation_id/test_facilities.py

```diff
@@ -152,6 +152,8 @@ def test_unapproved_industry_users_cannot_create_new_facility(self):
 
     # POST
     def test_post_new_malformed_facility(self):
+        owning_operator = operator_baker()
```

Comment:
Is this change because in the old implementation, we'd hit the 422 before the 401?

Reply:
Previously, data validation occurred before user validation, but now user validation takes precedence over data validation so we need to pass that part to be able to get to the actual test.